src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -156,7 +163,8 @@ export class WdsDataTableProvider implements DataTableProvider {
       supportsTypeDeletion: true,
       supportsTypeRenaming: false,
       supportsEntityRenaming: false,
-      supportsEntityUpdating: false, // TODO: enable as part of AJ-594
+      supportsEntityUpdating: true,
```

Comment:
If you rebase onto main (and, with apologies, resolve merge conflicts), you will hopefully get some changes to this file which showcase how I used capabilities backend to dynamically toggle these `supports` flags.  I'm guessing we may want to do something similar for edit attribute.

Reply:
the way I understand the capabilities bit is: it is possible someone may have a version of WDS that doesnt have a specific API call (like delete or rename), so we need to track this. 

since no backend calls were added to support editing (that API has been there from the start) -- it sounds like this shouldnt be impacted by this. But let me know if I am misunderstanding

Reply:
chatted more about this with Josh offline - capabilities needed for edit are all already supported, so confirmed that we do no need this for edit. Primarily this is the case because in WDS the edit modal will not be able to support a change of type like it does in GCP - in GCP there is no concept of the whole column being one type (each value in each individual column can end up being a different type, there is no strict typing per column), while in WDS that is required.